module04/01-Cash-Desk/solution.py

```diff
@@ -0,0 +1,79 @@
+"""Solution of module 4.1 - The Cash Desk problem"""
+class Bill:
```

Comment:
Old style class, please inherit from object => `Bill(object)`
https://docs.python.org/2/reference/datamodel.html#new-style-and-classic-classes